.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "8"
+  - "9"
```

Comment:
why do we need to update to version 9?

Reply:
Because some important dependencies (like mocha) are dropping support for older versions of nodejs, and in general we should continue to bump this number from time to time to make sure that we're not testing against outdated versions of nodejs. There's no reason to support them.

Reply:
Hum, so why the recommended version is still 8? https://nodejs.org/en/